typescript/utils/src/objects.ts

```diff
@@ -175,3 +175,27 @@ export function normalizeConfig(obj: WithAddress<any>): any {
 
   return obj;
 }
+
+// write a function that will go through an object and sort any arrays it finds
+export function sortValidatorsInConfig(config: any): any {
```

Comment:
@paulbalaji - we could extend this to sort modules in `staticAggregationIsm` by the ISM type